package.json

```diff
@@ -2,9 +2,11 @@
   "name": "@justeat/fozzie",
   "title": "Fozzie – Just Eat UI Web Framework",
   "description": "UI Web Framework for the Just Eat Global Platform",
-  "version": "0.30.0",
+  "version": "0.32.0",
+  "main": "dist/js/index.js",
```

Comment:
This needs to be `fozzie.js` now right?

Reply:
Nope – it's a bit confusing at the moment as we have two resolved entry points.

`fozzie.js` is the name of the file the gulp build will create.  `index.js` is the file used as the entry point when imported as a module.

Reply:
Gotcha.